showcase/src/app/demo-rbac/pages/authorization-directives/demo-authorization-directives-page.component.html

```diff
@@ -19,11 +20,14 @@ <h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
 
 			<div *starkShowOnPermission="onShowPermissionsConfig" class="protected-content">
 				{{ "SHOWCASE.DEMO_RBAC.DIRECTIVES.AUTHORIZATION.CONTENT_PROTECTED" | translate }}<br />
-				<b>{{ "SHOWCASE.DEMO_RBAC.DIRECTIVES.AUTHORIZATION.SHOW_ON_PERMISSION.ONLY_VISIBLE_TO" | translate: { role: "manager" } }}</b>
+				<b>{{
+					"SHOWCASE.DEMO_RBAC.DIRECTIVES.AUTHORIZATION.SHOW_ON_PERMISSION.ONLY_VISIBLE_TO" | translate: { role: "manager" }
```

Comment:
Could you put the brackets on the same line as the string here? eg: `{{ "..." }}`